src/storage/exec/ScanNode.h

```diff
@@ -42,55 +42,43 @@ class ScanVertexPropNode : public QueryNode<Cursor> {
   }
 
   nebula::cpp2::ErrorCode doExecute(PartitionID partId, const Cursor& cursor) override {
+    partId_ = partId;
     auto ret = RelNode::doExecute(partId);
     if (ret != nebula::cpp2::ErrorCode::SUCCEEDED) {
       return ret;
     }
 
     std::string start;
-    std::string prefix = NebulaKeyUtils::tagPrefix(partId);
+    std::string vertexPrefix = NebulaKeyUtils::vertexPrefix(partId);
```

Comment:
I doubt whether we should scan in this way, because it converts from one sequential scan to one sequential scan + lots of point look-up. Is there any other reason we need to scan in this way?

Reply:
May be we can san with two iterator.